site2/website/release-notes.md

```diff
@@ -3,6 +3,15 @@
 
 ### 2.8.0 &mdash; 2021-06-12 <a id=“2.8.0”></a>
 
+### Update notice
+
+Due to a breaking change in the Schema API it may happen that some Pulsar Functions or Pulsar IO Connector fail to work, 
+throwing an `IncompatibleClassChangeError`.
+In this case you have to rebuild your Function using Apache Pulsar 2.8.0 as dependency and redeploy it.
+If you are running on Kubernetes you can temporarily let the Functions Worker pod run with a previous version of Pulsar
+in order to not cause downtime.
+More context in [this issue](https://github.com/apache/pulsar/issues/11338).
```

Comment:
```suggestion
Due to a breaking change in the Schema API, it may happen that some Pulsar Functions or Pulsar IO connectors fail to work, 
throwing an `IncompatibleClassChangeError`.

In this case, you need to rebuild your Function using Apache Pulsar 2.8.0 as a dependency and redeploy it.

If you are running on Kubernetes, you can temporarily let the Functions Worker pod run with a previous version of Pulsar
in order to not cause downtime.

For more context about this issue, see [[Pulsar Functions and IO] Cannot upgrade Function built for Pulsar 2.7 to Pulsar 2.8](https://github.com/apache/pulsar/issues/11338).
```

1. can you give more context about `breaking change in the Schema API`? (for example, add a related link?)

Reply:
I added the link to https://github.com/apache/pulsar/pull/10878